### PR TITLE
Fix res string or bool issue, update depency

### PR DESCRIPTION
**Pull request by EARodriguezM**
Firebase_core dependency update due crash issue.
When the app is executed, if res is List<String>? This causes an exception, so it is better to use List<dynamic>? instead.

### CODE REVIEW

File: lib/features/home/presentation/utils/order_dialog_helper.dart [hunk 1]
```diff
@@ -58,7 +58,7 @@ abstract class OrderDialogHelper {
         message: message,
       ),
     ).then((result) {
-      final res = result as List<String>?;
+      final res = result as List<dynamic>?;
```
Comment (AhmedLSayed9) [inline lib/features/home/presentation/utils/order_dialog_helper.dart:61]: It should be List<bool> instead.
maybe just returning a bool will be better too

**Comment by AhmedLSayed9** [comment]
> Firebase_core dependency update due crash issue.

Can you elaborate which crash issue ?

File: lib/features/home/presentation/utils/order_dialog_helper.dart (hunk 1 above)
Comment (EARodriguezM) [reply lib/features/home/presentation/utils/order_dialog_helper.dart:61]: You are right.

**Comment by EARodriguezM** [comment]
I solved these problems days ago and until now I remember this so I sent the pull. After tracing, there were issues running the flutterfire configure command, which were solved by updating the dependency.

**Comment by AhmedLSayed9** [comment]
Thanks!

**Pull request merged**